JavaAssignment/src/rashmi/LargestOf3.java

```diff
@@ -0,0 +1,29 @@
+package rashmi;
+
+import java.util.Scanner;
+
+public class LargestOf3 {
+   
+	void findLarge(int num1,int num2,int num3)
+	{
+		if(num1 > num2 && num1 > num3)
+			System.out.println(num1);
+		
+		else if(num2 > num3 && num2 > num1)
```

Comment:
Here no need to compare again num1 & num2. Think here, 
num1>num2(Line 1)
num2>num2(Line 2)